### PR TITLE
marija-mitreska login

### PR DESCRIPTION
**Pull request by marijamitreska**

### CODE REVIEW

File: Project/demo/src/main/java/com/example/demo/controller/AuthenticationController.java [hunk 1]
```diff
@@ -52,11 +55,21 @@ public String login(Model model) {
         return "login_user";
     }
 
-    @GetMapping("/users")
-    public String listUsers(Model model) {
+    @RequestMapping(path = "/login", method = RequestMethod.POST)
+    @ResponseBody
+    public ResponseEntity<String> homePage(@RequestBody UserDTO userdto) {
```
Comment (aleksandar-bujaroski-armedia) [inline Project/demo/src/main/java/com/example/demo/controller/AuthenticationController.java]: Change variable name from userdto to userDTO

File: Project/demo/src/main/java/com/example/demo/dto/UserDTO.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.example.demo.dto;
+
+import javax.persistence.Column;
+
+public class UserDTO {
+    private String email;
```
Comment (aleksandar-bujaroski-armedia) [inline Project/demo/src/main/java/com/example/demo/dto/UserDTO.java:4]: Fix code formatting in the whole project

**Review by aleksandar-bujaroski-armedia — CHANGES_REQUESTED** [review]

File: Project/demo/src/main/java/com/example/demo/service/AuthenticationServiceImpl.java [hunk 3]
```diff
@@ -67,6 +75,32 @@ public void saveUser(User user) {
         useRepo.save(user);
     }
 
+    @Override
+    public String validateUserForLogin(UserDTO user) {
```
Comment (aleksandar-bujaroski-armedia) [inline Project/demo/src/main/java/com/example/demo/service/AuthenticationServiceImpl.java]: Fix code formatting

File: Project/demo/src/main/java/com/example/demo/utill/PasswordEncoder.java [hunk 4]
```diff
@@ -5,7 +5,7 @@
 public class PasswordEncoder {
```
Comment (aleksandar-bujaroski-armedia) [inline Project/demo/src/main/java/com/example/demo/utill/PasswordEncoder.java]: Fix code formatting

**Review by aleksandar-bujaroski-armedia — CHANGES_REQUESTED** [review]

**Review by dejannastevski-armedia — APPROVED** [review]

**Pull request merged**